src/validators/account-number/account-number-validator.ts

```diff
@@ -0,0 +1,9 @@
+import { mod11 } from '../../utils/mod11';
+
+export const isValidAccountNumber = (accountNumber: string): boolean => {
```

Comment:
Burde vi på en måte gjøre det eksplisitt at vi er i kontekst av Norge her?

Vet ikke hva beste løsning for det er, men kanskje putte norge-spesifikke ting i en mappe "norway", eller rename funksjonen til `isValidNorwegianAccountNUmber` ?

Reply:
@thrap oppdaterer du og merger?